src/stravalib/tests/unit/test_model.py

```diff
@@ -35,12 +36,6 @@ def test_legacy_deserialize(self, model_class, attr, value):
             model_obj = model_class.deserialize({attr: value})
             assert getattr(model_obj, attr) == value
 
-    def test_legacy_from_dict(self, model_class, attr, value):
```

Comment:
we could potentially remove all of these legacy tests and methods (it's only 3 i think in that mixin). we could also do that in a separate PR to allow for smaller pr's (this one is already big!).

Reply:
It's ok to remove them now.